CMakeLists.txt

```diff
@@ -290,6 +290,9 @@ CMAKE_DEPENDENT_OPTION(
   "Slicer_BUILD_ITKPython" OFF)
 mark_as_superbuild(Slicer_INSTALL_ITKPython)
 
+option(Slicer_BUILD_PARAMETERSERIALIZER_SUPPORT "Build Slicer with parameter serializer support" OFF)
```

Comment:
@lassoan: @cpinter raised this issue earlier.

As @jcfr explained in his response at the time: "As of today, VTK dependencies are internal and not intended to be used externally."